woocommerce/woocommerce
# Change select option hover color on core profiler pages

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This is a follow-up PR of #38709 to address the select option hover color that was not included in the original PR.

I also pushed an unrelated [fix](https://github.com/woocommerce/woocommerce/pull/38724/commits/bcc8f213f5d69d1fe436fe103b85f204679fd5a5) in this PR as it's small enough to include. It fixes the continue button position on the plugins page on mobile view.



### How to test the changes in this Pull Request:


1. Install and activate WooCommerce on a brand new site
1. Install the WooCommerce Beta Tester plugin
1. Go to /wp-admin/tools.php?page=woocommerce-admin-test-helper
1. Enable the 'core-profiler' feature flag
1. Visit `/wp-admin/admin.php?page=wc-admin&path=%2Fsetup-wizard`
1. Go to `Which one of these best describes you?` page and select `I'm already selling'.
2. Click `Select an option` and confirm the hover color.
3. Click `Continue`
4. Click dropdowns and confirm the hover color.
